### PR TITLE
[es] add concepts/storage/volumes

### PR DESCRIPTION
**Pull request by anyulled**
Related the issue #13948

### CODE REVIEW

File: content/es/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -0,0 +1,1246 @@
...
+{{< note >}}
+
+- Debes crear un [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) antes de usarlo.
+- Un contenedor usando un ConfigMap montado como un volumen [`subPath`](#using-subpath) no recibirá actualizaciones del ConfigMap
+- Los datos de texto son expuestos como ficheros usando la codificación de caracteres UTF-8. Para otras codificaciones de caracteres, use `binaryData`.
+{{< /note >}}
+
+### downwardAPI {#downwardapi}
+
+Un volumen de `downwardAPI` hace que los datos API descendentes estén disponibles para las aplicaciones.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:231]:
```suggestion
Un volumen de `downwardAPI` hace que los datos descendentes de la API estén disponibles para las aplicaciones.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -0,0 +1,1246 @@
...
+Monta un directorio y escribe los datos solicitados en archivos de texto sin formato.
+
+{{< note >}}
+Un contenedor usando la API descendiente montado como un volumen [`subPath`](#using-subpath) no recibirá actualizaciones API descendientes.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:235]:
```suggestion
Un contenedor usando los datos descendentes de la API montadolos como un volumen [`subPath`](#using-subpath) no recibirá actualizaciones de la API.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 3]
```diff
@@ -0,0 +1,1246 @@
...
+{{< /note >}}
+
+Mira el [downward API example](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) para mayores detalles.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:238]:
```suggestion
Mira el [ejemplo de downward API](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) para mayores detalles.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 4]
```diff
@@ -0,0 +1,1246 @@
...
+
+### emptyDir {#emptydir}
+
+Un volumen `emptyDir`es creado primero cuando se asigna un pod a un nodo, y existe mientras el Pod está corriendo en el nodo.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:242]:
```suggestion
Un volumen `emptyDir`es creado primero cuando se asigna un Pod a un nodo, y existe mientras el Pod está corriendo en el nodo.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 5]
```diff
@@ -0,0 +1,1246 @@
...
+Como su nombre lo indica un volumen `emptydir`está vacío inicialmente. Todos los contenedores en el Pod pueden leer y escribir
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:243]:
```suggestion
Como su nombre lo indica un volumen `emptydir`está inicialmente vacío. Todos los contenedores en el Pod pueden leer y escribir
```

File: content/es/docs/concepts/storage/volumes.md [hunk 6]
```diff
@@ -0,0 +1,1246 @@
...
+los mismos ficheros en el volumen `emptyDir`, aunque ese volumen se puede montar en la misma ruta o diferentes en cada contenedor. Cuando un Pod es removido del nodo por alguna razón, los datos en `emptydir` se borran permanentemente.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:244]:
```suggestion
los archivos en el volumen `emptyDir`, aunque ese volumen se puede montar en la misma o diferente ruta en cada contenedor. Cuando un Pod es removido del nodo por alguna razón, los datos en `emptydir` se borran permanentemente.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 7]
```diff
@@ -0,0 +1,1246 @@
...
+{{< /note >}}
+
+Algunos usos para un `emptyDir` son:
+
+- Espacio temporal, como para una clasificación de combinación basada en disco
+- Marcar un largo cálculo para la recuperación de fallos
+- Contener archivos que un contenedor de administrador de contenido recupera mientras un contenedor de servidor web
+  sirve los datos
+
+Dependiendo de tu entorno, volúmenes `eptydir` se almacenan en cualquier medio que respalde el nodo tales como disco SSD, o almacenamiento de red. Sin embargo, si estableces el campo `emptydir.medium` a `Memory`, Kubernetes monta en su lugar un tmpfs (sistema de ficheros respaldado por la RAM). Mientras que tmpfs es muy rápido, ten en cuenta que a diferencia de los discos, tmpfs se limpia cuando el nodo reinicia y cualquier fichero que escribas cuenta contra el límite de memoria del contenedor.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:258]:
```suggestion
Dependiendo de tu entorno, los volúmenes `emptydir` se almacenan en cualquier medio que respalde el nodo tales como disco SSD, o almacenamiento de red. Sin embargo, si se establece el campo `emptydir.medium` a `Memory`, Kubernetes monta en su lugar un tmpfs (sistema de ficheros respaldado por la RAM). Mientras que tmpfs es muy rápido, ten en cuenta que a diferencia de los discos, tmpfs se limpia cuando el nodo reinicia y cualquier archivo que escribas cuenta con el límite de memoria del contenedor.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 8]
```diff
@@ -0,0 +1,1246 @@
...
+
+{{< note >}}
+Si el [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) `SizeMemoryBackedVolumes` está habilitado,
+puedes especificar un tamaño para los volúmenes respaldados en memoria. Si no se especifica ningún tamaño los volúmenes respaldados en memoria tienen un tamaño del 50% de la memoria en un host Linux.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:262]:
```suggestion
puedes especificar un tamaño para los volúmenes respaldados en memoria. Si no se especifica ningún tamaño, los volúmenes respaldados en memoria tienen un tamaño del 50% de la memoria en un host Linux.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 9]
```diff
@@ -0,0 +1,1246 @@
...
+          name: cache-volume
+  volumes:
+    - name: cache-volume
+      emptyDir: {}
+```
+
+### fc (canal de fibra) {#fc}
+
+Un tipo de volumen `fc` permite que un volumen de almacenamiento de bloque de canal de fibra existente se monte en un Pod.
+Puede especificar nombres mundiales de destino únicos o múltiples (WWN) utilizando el parámetro `targetWWNs` en su configuración de Volumen.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:287]:
```suggestion
Puede especificar nombres mundiales de destino únicos o múltiples (WWN) utilizando el parámetro `targetWWNs` en su configuración de volumen.
```

**Review by electrocucaracha — COMMENTED** [review]
Envio algunas observaciones del documento, continuare con la revisión del resto en estos dias

File: content/es/docs/concepts/storage/volumes.md [hunk 10]
```diff
@@ -0,0 +1,1246 @@
...
+{{< note >}}
+Debes tener una instalación propia de Flocker ejecutándose antes de poder usarla.
+{{< /note >}}
+
+Mira el [ejemplo de Flocker ](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/flocker) para más detalles.
+
+### gcePersistentDisk
+
+Un volumen `gcePersistentDisk` monta un volumen de Google Compute Engine (GCE)
+[persistent disk](https://cloud.google.com/compute/docs/disks) (DP) en tu Pod.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:314]:
```suggestion
de [disco persistente](https://cloud.google.com/compute/docs/disks) (DP) en tu Pod.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 11]
```diff
@@ -0,0 +1,1246 @@
...
+A diferencia de `emptyDir`, que se borra cuando el pod es removido, el contenido de un DP es preservado
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:315]:
```suggestion
A diferencia de `emptyDir`, que se borra cuando el pod es removido, el contenido de un disco persistente es preservado
```

File: content/es/docs/concepts/storage/volumes.md [hunk 12]
```diff
@@ -0,0 +1,1246 @@
...
+y el volumen solamente se desmonta. Esto significa que un PD puede ser pre-poblado con datos,
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:316]:
```suggestion
y el volumen solamente se desmonta. Esto significa que un disco persistente puede ser pre-poblado con datos,
```

File: content/es/docs/concepts/storage/volumes.md [hunk 13]
```diff
@@ -0,0 +1,1246 @@
...
+y que esos datos se pueden compartir entre pods.
+
+{{< note >}}
+Debes crear un PD usando `gcloud` or la API de GCE o la UI antes de poder usarlo.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:320]:
```suggestion
Debes crear un disco persistente usando `gcloud`, la API de GCE o la UI antes de poder usarlo.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 14]
```diff
@@ -0,0 +1,1246 @@
...
+{{< /note >}}
+
+Existen algunas restricciones cuando usas `gcePersistentDisk`:
+
+- Los nodos en los que se ejecutan los pods deben ser máquinas virtuales GCE.
+- Esas máquinas virtuales deben estar en el mismo proyecto GCE y zona que el disco persistente.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:326]:
```suggestion
- Esas máquinas virtuales deben estar en el mismo proyecto GCE y zona que el disco persistente se encuentra.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 15]
```diff
@@ -0,0 +1,1246 @@
...
+
+Una de las características del disco persistente CGE es acceso concurrente de solo lectura al disco persistente.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:328]:
```suggestion
Una de las características del disco persistente GCE es acceso concurrente de solo lectura al disco persistente.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 16]
```diff
@@ -0,0 +1,1246 @@
...
+Un volumen `gcePersistentDisk` permite montar simultáneamente un disco de solo lectura a múltiples consumidores.
+Esto significa que puedes pre-poblar un DP con tu conjunto de datos y luego servirlo en paralelo desde tantos pods como necesites. Desafortunadamente, los DPs solo se pueden montar por un único consumidor en modo lectura-escritura.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:330]:
```suggestion
Esto significa que puedes pre-poblar un disco persistente con tu conjunto de datos y luego servirlo en paralelo desde tantos pods como necesites. Desafortunadamente, los discos persistentes solo se pueden montar en un único consumidor en modo lectura-escritura.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 17]
```diff
@@ -0,0 +1,1246 @@
...
+No están permitidos escritores simultáneos.
+
+Usar un disco persistente GCE con un Pod controlado por un ReplicaSet fallará a manos que el DP sea de solo lectura o el número de réplicas sea 0 o 1.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:333]:
```suggestion
Usar un disco persistente GCE con un Pod controlado por un ReplicaSet fallará a menos que el disco persistente sea de solo lectura o el número de réplicas sea 1 o 0.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 18]
```diff
@@ -0,0 +1,1246 @@
...
+spec:
+  containers:
+    - image: k8s.gcr.io/test-webserver
+      name: test-container
+      volumeMounts:
+        - mountPath: /test-pd
+          name: test-volume
+  volumes:
+    - name: test-volume
+      # Este PD GCE debe existir con anterioridad.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:359]:
```suggestion
      # Este disco persistente GCE debe existir con anterioridad.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 19]
```diff
@@ -0,0 +1,1246 @@
...
+      gcePersistentDisk:
+        pdName: my-data-disk
+        fsType: ext4
+```
+
+#### Discos regionales persistentes
+
+La función de [discos regionales persistentes](https://cloud.google.com/compute/docs/disks/#repds)
+permite la creación de discos persistentes que están disponibles en dos zonas dentro de la misma región.
+Para usar esta función, el volumen debe ser provisto como un PersistentVolumen; referenciar el volumen directamente desde un pod no está soportado.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:369]:
```suggestion
Para usar esta función, el volumen debe ser provisto como un PersistentVolumen; referenciar el volumen directamente desde un Pod no está soportado.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 20]
```diff
@@ -0,0 +1,1246 @@
...
+
+#### Aprovisionamiento manual de un PD PersistentVolume Regional
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:371]:
```suggestion
#### Aprovisionamiento manual de un disco regional persistente PersistentVolume
```

File: content/es/docs/concepts/storage/volumes.md [hunk 21]
```diff
@@ -0,0 +1,1246 @@
...
+              values:
+                - us-central1-a
+                - us-central1-b
+```
+
+#### Migración CSI GCE
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+La función `CSIMigration` para el DP GCE, cuando se habilita, redirige todas las operaciones
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:412]:
```suggestion
La función `CSIMigration` para el disco persistente GCE, cuando se habilita, redirige todas las operaciones
```

File: content/es/docs/concepts/storage/volumes.md [hunk 22]
```diff
@@ -0,0 +1,1246 @@
...
+de complemento desde el complemento existente dentro del árbol existente al controlador de Interfaz de Almacenamiento del Contenedor (CSI) de `pd.csi.storage.gke.io`. Para poder usar esta función, el [controlador PD GCE](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) debe ser instalado en el clúster y habilitar las funciones beta
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:413]:
```suggestion
de complemento desde el complemento existente dentro del árbol existente al controlador de Interfaz de Almacenamiento del Contenedor (CSI) de `pd.csi.storage.gke.io`. Para poder usar esta función, el [controlador de disco persistentes GCE](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) debe ser instalado en el clúster y habilitar las funciones beta
```

File: content/es/docs/concepts/storage/volumes.md [hunk 23]
```diff
@@ -0,0 +1,1246 @@
...
+    - name: git-volume
+      gitRepo:
+        repository: "git@somewhere:me/my-git-repository.git"
+        revision: "22f1d8406d464b0c0874075539c1f2e96c253775"
+```
+
+### glusterfs
+
+Un volumen `glusterfs` permite montar un volumen [Glusterfs](https://www.gluster.org) en tu Pod.
+A diferencia de `emptyDir`, que se borra cuando se remueve un pod, el contenido de un volumen `glusterfs` es preservado
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:448]:
```suggestion
A diferencia de `emptyDir`, que se borra cuando se remueve un Pod, el contenido de un volumen `glusterfs` es mantenido 
```

File: content/es/docs/concepts/storage/volumes.md [hunk 24]
```diff
@@ -0,0 +1,1246 @@
...
+
+{{< note >}}
+Debes tener tu propia instalación de GlusterFS ejecutándose antes de poder usarla.
+{{< /note >}}
+
+Mira el [ejemplo de GlusterFS](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/volumes/glusterfs) para más detalles.
+
+### hostPath {#hostpath}
+
+Un volumen `hostPath` mona un fichero o un directorio del sistema de archivos del nodo host a tu Pod.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:460]:
```suggestion
Un volumen `hostPath` monta un archivo o directorio del sistema de archivos del nodo Host a tu Pod.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 25]
```diff
@@ -0,0 +1,1246 @@
...
+Esto no es algo de muchos Pods necesiten, pero ofrece una trampa de escape poderosa para algunas aplicaciones.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:461]:
```suggestion
Esto no es algo de muchos Pods necesiten, pero ofrece un atajo poderoso para algunas aplicaciones.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 26]
```diff
@@ -0,0 +1,1246 @@
...
+
+Por ejemplo, algunos usos de un `hostPath` son:
+
+- ejecutar un contenedor que necesita acceso a los directorios internos de docker, usa un `hostPath` de `/var/lib/docker`
+- ejecutar un cAdvisor en un contenedor; usa un `hostPath` de `/sys`
+- permitir a un Pod especificar si un `hostPath` dado debería existir ante de correr el Pod, si debe crearse, cómo debe existir
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:467]:
```suggestion
- permitir a un Pod especificar si un `hostPath` dado debería existir antes de correr el Pod, y si debe crearse, como este debe existir
```

File: content/es/docs/concepts/storage/volumes.md [hunk 27]
```diff
@@ -0,0 +1,1246 @@
...
+
+Además de la propiedad requerida `path`, puedes especificar opcionalmente un `tipo`para un volumen `hostPath`.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:469]:
```suggestion
Además de la propiedad requerida `path`, puedes especificar opcionalmente un `tipo` para un volumen `hostPath`.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 28]
```diff
@@ -0,0 +1,1246 @@
...
+
+Los valores soportados para el campo `tipo` son:
+
+| Valor               | Comportamiento                                                                                                                                                   |
+| :------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                     | Una cadena vacía (por defecto) es para compatibilidad con versiones anteriores, lo que significa que no se harán revisiones antes de montar el volumen hostPath. |
+| `DirectoryOrCreate` | Si no hay nada en la ruta dada, se creará un directorio vacío como es requerido con los permisos a 0755, teniendo el mismo grupo y propiedad que el Kubelet.     |
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:476]:
```suggestion
| `DirectoryOrCreate` | Si no hay nada en la ruta dada, se creará un directorio vacío como es requerido con los permisos a 0755, teniendo el mismo grupo y propiedad que Kubelet.     |
```

File: content/es/docs/concepts/storage/volumes.md [hunk 29]
```diff
@@ -0,0 +1,1246 @@
...
+| `Directory`         | Un directorio debe existir en la ruta dada                                                                                                                       |
+| `FileOrCreate`      | Si no hay nada en la ruta dada, se creará un fichero vacío como es requerido con los permisos a 0644, teniendo el mismo grupo y propiedad que el Kubelet.        |
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:478]:
```suggestion
| `FileOrCreate`      | Si no hay nada en la ruta dada, se creará un archivo vacío como es requerido con los permisos a 0644, teniendo el mismo grupo y propiedad que Kubelet.        |
```

File: content/es/docs/concepts/storage/volumes.md [hunk 30]
```diff
@@ -0,0 +1,1246 @@
...
+| `File`              | Un fichero debe existir en la ruta dada                                                                                                                          |
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:479]:
```suggestion
| `File`              | Un archivo debe existir en la ruta dada                                                                                                                          |
```

File: content/es/docs/concepts/storage/volumes.md [hunk 31]
```diff
@@ -0,0 +1,1246 @@
...
+| `Socket`            | Un socket de UNIX debe existir en la ruta dada                                                                                                                   |
+| `CharDevice`        | Un dispositivo de caracteres debe existir en la ruta data                                                                                                        |
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:481]:
```suggestion
| `CharDevice`        | Un dispositivo de caracteres debe existir en la ruta dada                                                                                                        |
```

File: content/es/docs/concepts/storage/volumes.md [hunk 32]
```diff
@@ -0,0 +1,1246 @@
...
+| `BlockDevice`       | Un dispositivo de bloques dbe existir en la ruta dada                                                                                                            |
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:482]:
```suggestion
| `BlockDevice`       | Un dispositivo de bloques debe existir en la ruta dada                                                                                                            |
```

File: content/es/docs/concepts/storage/volumes.md [hunk 33]
```diff
@@ -0,0 +1,1246 @@
...
+
+Ten cuidado cuando uses este tipo de volumen, porque:
+
+- Los Pods con configuración idéntica (tales como los creados por un PodTemplate) pueden comportarse de forma distinta
+  en nodos distintos debido a diferentes ficheros en los nodos.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:487]:
```suggestion
  en nodos distintos debido a diferentes archivos en los nodos.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 34]
```diff
@@ -0,0 +1,1246 @@
...
+- Los ficheros o directorios creados en los hosts subyacentes son modificables solo por root. Debes ejecutar tu proceso como root en un [Contenedor privilegiado](/docs/tasks/configure-pod-container/security-context/) o modificar los permisos de archivo en el host para escribir a un volumen `hostPath`
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:488]:
```suggestion
- Los archivos o directorios creados en los Hosts subyacentes son modificables solo por root. Debes ejecutar tu proceso como root en un [Contenedor privilegiado](/docs/tasks/configure-pod-container/security-context/) o modificar los permisos de archivo en el Host para escribir a un volumen `hostPath`
```

File: content/es/docs/concepts/storage/volumes.md [hunk 35]
```diff
@@ -0,0 +1,1246 @@
...
+    - name: test-volume
+      hostPath:
+        # localización del directorio en el host
+        path: /data
+        # este campo es opcional
+        type: Directory
+```
+
+{{< caution >}}
+El modo `FileOrCreate` no crea el directorio padre del fichero. Si el directorio padre del fichero montado no existe,
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:514]:
```suggestion
El modo `FileOrCreate` no crea el directorio padre del archivo. Si el directorio padre del archivo montado no existe,
```

File: content/es/docs/concepts/storage/volumes.md [hunk 36]
```diff
@@ -0,0 +1,1246 @@
...
+el pod falla en iniciar. Para asegurar que este modo funciona, puedes intentar montar directorios y ficheros de forma separada,
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:515]:
```suggestion
el Pod falla en iniciar. Para asegurar que este modo funciona, puedes intentar montar directorios y archivos de forma separada,
```

File: content/es/docs/concepts/storage/volumes.md [hunk 37]
```diff
@@ -0,0 +1,1246 @@
...
+      image: k8s.gcr.io/test-webserver:latest
+      volumeMounts:
+        - mountPath: /var/local/aaa
+          name: mydir
+        - mountPath: /var/local/aaa/1.txt
+          name: myfile
+  volumes:
+    - name: mydir
+      hostPath:
+        # Asegúrate que el directorio del fichero es creado.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:538]:
```suggestion
        # Asegúrate que el directorio del archivo es creado.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 38]
```diff
@@ -0,0 +1,1246 @@
...
+      hostPath:
+        path: /var/local/aaa/1.txt
+        type: FileOrCreate
+```
+
+### iscsi
+
+Un volumen `iscsi` permite que se monte un volumen ISCSI (SCSI sobre IP) existente
+en tu Pod. A diferencia de `emptydir`, que es removido cuando se remueve un Pod,
+el contenido de un volumen `iscsi` es preservado y el volumen solamente se desmonta.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:551]:
```suggestion
el contenido de un volumen `iscsi` es mantenido y el volumen solamente se desmonta.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 39]
```diff
@@ -0,0 +1,1246 @@
...
+Esto significa que un volumen iscsi puede ser pre-poblado con dato, y que estos datos se pueden compartir entre pods.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:552]:
```suggestion
Esto significa que un volumen iscsi puede ser pre-poblado con datos, y que estos datos se pueden compartir entre pods.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 40]
```diff
@@ -0,0 +1,1246 @@
...
+{{< note >}}
+Debes tener tu propio servidor ISCSI corriendo con el volumen creado antes de poder usarlo.
+{{< /note >}}
+
+Una función de SCSI es que puede ser montado como de solo lectura por múltiples consumidores simultáneamente.
+Esto significa que puedes pre-poblar un volumen con tu conjunto de datos y servirlo en paralelo para tantos Pods como necesites.
+Desafortunadamente, los volúmenes ISCSI solo se pueden montar por un único consumidor en modo lectura-escritura.
+Escritores simultáneos no está permitido.
+
+Mira el [ ejemplo iSCSI](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/volumes/iscsi) para más detalles.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:563]:
```suggestion
Mira el [ejemplo iSCSI](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/volumes/iscsi) para más detalles.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 41]
```diff
@@ -0,0 +1,1246 @@
...
+
+Un volumen `local` representa un dispositivo de almacenamiento local como un disco, una partición o un directorio.
+
+Los volúmenes locales solo se pueden usar como un PersistenVolume creado estáticamente.
+El aprovisionamiento dinámico no está soportado.
+
+Comparados con volúmenes `hostPath`, los volúmenes `local` se usan de manera duradera y portátil sin programar pods manualmente
+a los nodos. El sistema está consciente de las limitaciones del nodo del volumen al mirar la afinidad del nodo en el PersistenVolumen.
+
+Sn embargo, los volúmenes `local`están sujetos a la disponibilidad del nodo subyacente y no son compatibles para todas las aplicaciones.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:575]:
```suggestion
Sin embargo, los volúmenes `local`están sujetos a la disponibilidad del nodo subyacente y no son compatibles para todas las aplicaciones.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 42]
```diff
@@ -0,0 +1,1246 @@
...
+
+Si un nodo deja de estar sano, entonces el volumen `local` se vuelve inaccesible al pod.
+El pod que utiliza este volumen no se puede ejecutar.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:578]:
```suggestion
El Pod que utiliza este volumen no se puede ejecutar.
```

File: content/es/docs/concepts/storage/volumes.md [hunk 43]
```diff
@@ -0,0 +1,1246 @@
...
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: kubernetes.io/hostname
+              operator: In
+              values:
+                - example-node
+```
+
+Debes establecer un valor de `nodeAffinity` del PersistenVolume cuando uses volúmenes `local`.
+El programador de Kubernetes usa esta `nodeaffinity` del PersistenVolume para programar estos Pods al nodo correcto.
```
Comment (electrocucaracha) [inline content/es/docs/concepts/storage/volumes.md:610]:
```suggestion
El Scheduler de Kubernetes usa `nodeaffinity` del PersistenVolume para programar estos Pods al nodo correcto.
```

**Comment by anyulled** [comment]
Hi @raelga @electrocucaracha any progress on this?

**Comment by raelga** [comment]
Hey @Anynou, sorry for the delay and thanks for the updates.

/lgtm
/approve

**Comment by Anynou** [comment]
Don't worry.

**Comment by anyulled** [comment]
Thanks @raelga!

**Pull request merged**